Monika After Story/game/script-greetings.rpy

```diff
@@ -691,3 +691,25 @@ label greeting_sunshine:
     m 1l "I was just singing to myself to pass time."
     m 1b "But now that you're here, we can spend some time together."
     return
+
+if renpy.seen_event("monika_algernon"):
```

Comment:
oh shit wait `seen_event` is a custom function written by us, not a renpy function. You should use `seen_label` here instead.